tests/devices/test_default_qutrit.py

```diff
@@ -797,15 +830,171 @@ def test_hermitian_two_wires_identity_expectation(self, tol):
         expected = 3.5 * 1 * 1
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
+    @pytest.mark.parametrize(
+        "index_1, eval_1", list(zip(list(range(1, 9)), [0, 0, -1, 0, 0, 0, 0, 1 / math.sqrt(3)]))
+    )
+    @pytest.mark.parametrize(
+        "index_2, eval_2", list(zip(list(range(1, 9)), [0, 0, 0, 0, 0, 0, 0, -2 / math.sqrt(3)]))
+    )
+    def test_gell_mann_tensor(self, index_1, index_2, eval_1, eval_2, tol):
+        """Test that the expectation value of the tensor product of two Gell-Mann observables is
+        correct"""
+        dev = qml.device("default.qutrit", wires=2)
+        obs = qml.GellMannObs(index_1, wires=0) @ qml.GellMannObs(index_2, wires=1)
+
+        dev.apply(
+            [
+                qml.QutritUnitary(TSHIFT, wires=0),
+                qml.QutritUnitary(TSHIFT, wires=1),
+                qml.QutritUnitary(TADD, wires=[0, 1]),
+            ],
+            obs.diagonalizing_gates(),
+        )
+        res = dev.expval(obs)
+
+        expected = eval_1 * eval_2
+        assert np.isclose(res, expected, atol=tol, rtol=0)
+
 
-# TODO: Add tests for tensor non-parametrized observables
 class TestTensorVar:
-    pass
+    """Tests for variance of tensor observables"""
+
+    @pytest.mark.parametrize("index_1", list(range(1, 9)))
+    @pytest.mark.parametrize("index_2", list(range(1, 9)))
+    def test_gell_mann_tensor(self, index_1, index_2, tol):
+        """Test that the variance of tensor Gell-Mann observables is correct"""
+        dev = qml.device("default.qutrit", wires=2)
+        obs = qml.GellMannObs(index_1, wires=0) @ qml.GellMannObs(index_2, wires=1)
+
+        dev.apply(
+            [
+                qml.QutritUnitary(U_thadamard_01, wires=0),
+                qml.QutritUnitary(TSHIFT, wires=0),
+                qml.QutritUnitary(TSHIFT, wires=1),
+                qml.QutritUnitary(TADD, wires=[0, 1]),  # (|12> + |20>) / sqrt(2)
+            ],
+            obs.diagonalizing_gates(),
+        )
+        res = dev.var(obs)
+
+        state = np.array([[0, 0, 0, 0, 0, 1, 1, 0, 0]]) / math.sqrt(2)
+        obs_mat = np.kron(GELL_MANN[index_1 - 1], GELL_MANN[index_2 - 1])
+
+        expected = (
+            state.conj() @ obs_mat @ obs_mat @ state.T - (state.conj() @ obs_mat @ state.T) ** 2
+        )
+        assert np.isclose(res, expected[0], atol=tol, rtol=0)
+
+    def test_hermitian(self, tol):
```

Comment:
Missing docstring for this test; was it just missed from a previous PR when Hermitian was added?